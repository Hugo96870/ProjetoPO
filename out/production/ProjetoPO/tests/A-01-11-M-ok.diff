*** tests/A-01-11-M-ok.out	2020-11-10 08:52:30.000000000 +0000
--- tests/A-01-11-M-ok.outhyp	2020-11-21 19:45:42.123145000 +0000
***************
*** 22,28 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 10
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 22,28 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
