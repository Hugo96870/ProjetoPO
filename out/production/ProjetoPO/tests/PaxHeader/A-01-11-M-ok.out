30 mtime=1604998350.588095641
